dopamine_BL21/11/11_stats.jl

```diff
@@ -0,0 +1,140 @@
+
+
+
+# MODELS / ANOVAS
+
+# comparing across strains for buffer medium
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(==("M9"))))
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ────────────────────────────────────────────────────────────────────────────────────────
+#                                Coef.  Std. Error       t  Pr(>|t|)  Lower 95%  Upper 95%
+# ────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 292.938      12.4953   23.44    <1e-46   268.206    317.669
+# worm: CB                    -45.5792     15.6637   -2.91    0.0043   -76.582    -14.5764
+# worm: MT                    -69.0725     16.422    -4.21    <1e-04  -101.576    -36.5687
+# bacteria: BL21             -171.053      15.8812  -10.77    <1e-18  -202.487   -139.62
+# worm: CB & bacteria: BL21   149.003      22.0993    6.74    <1e-09   105.262    192.744
+# worm: MT & bacteria: BL21   123.34       22.0164    5.60    <1e-06    79.7631   166.916
+# ────────────────────────────────────────────────────────────────────────────────────────
+
+# CB on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of CB vs speed diff of N2
+# p = <1e-09 --> significant difference, as expected
+
+# MT on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of MT vs speed diff of N2
+# p = <1e-06 --> significant difference, as expected
+
+
+
+# comparing across strains for dopamine medium
+lm(@formula(meanspeed ~ worm*bacteria), subset(trackstats, :medium=>ByRow(==("DA"))))
+# meanspeed ~ 1 + worm + bacteria + worm & bacteria
+
+# Coefficients:
+# ───────────────────────────────────────────────────────────────────────────────────────────
+#                                 Coef.  Std. Error       t  Pr(>|t|)   Lower 95%   Upper 95%
+# ───────────────────────────────────────────────────────────────────────────────────────────
+# (Intercept)                 236.311       8.82898   26.77    <1e-57   218.862    253.76
+# worm: CB                      7.86592    11.9762     0.66    0.5123   -15.8032    31.5351
+# worm: MT                    -25.0427     12.2116    -2.05    0.0421   -49.177     -0.908308
+# bacteria: BL21             -118.027      11.3625   -10.39    <1e-18  -140.483    -95.5704
+# worm: CB & bacteria: BL21    -3.19201    16.1265    -0.20    0.8434   -35.0635    28.6795
+# worm: MT & bacteria: BL21    33.5566     16.2246     2.07    0.0404     1.49108   65.6221
+# ───────────────────────────────────────────────────────────────────────────────────────────
+
+# CB on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of CB vs speed diff of N2
+# p = 0.8434 --> not significant, as expected
+
+# MT on and off bacteria vs N2 on and off bacteria
+# ie. speed diff of MT vs speed diff of N2
+# p = 0.0404 --> BARELY SIGNIFICANT, KINDA EXPECTED?
```

Comment:
MT on and off bacteria vs N2 on and off bacteria
p = 0.0404 --> BARELY SIGNIFICANT, KINDA EXPECTED?